lib/server.js

```diff
@@ -20,7 +24,7 @@ const HANDSHAKE_TIMEOUT = 3000;
 //              server 'connect' event (optional).
 //
 const initServer = function(
```

Comment:
You should add sessionStorageProvider parameter to the description of the method.